telethon_generator/generators/docs.py

```diff
@@ -77,6 +77,17 @@ def _find_title(html_file):
     return '(Unknown)'
 
 
+def _find_if_bot_can_use(html_file):
```

Comment:
Ew ew ew. Don't do that. Do this:
https://github.com/LonamiWebs/Telethon/blob/0418f7e37553b167786b40d6554eda4e2cfe9152/telethon_generator/generators/docs.py#L269-L271

Reply:
but `_generate_index` only uses the generated html paths not the tlobjects?

Reply:
Good point. Maybe that should change? Relying on the order in which they're generated is definitely not good.